source/includes/examples/_energyplus.html.md

```diff
@@ -0,0 +1,259 @@
+
+## Inkling File
+
+###### Schema
+
+```inkling
+schema SimState
+    Int32{0:10} SolarIrradiation
+end
+```
+
+The `SimState` schema defines the dictionary returned from the Python simulation's advance method to the BRAIN including:
+   Double TOut,
+   Double TZone,
+   Double SolarIrradiation
+   Double FractionShadingOn
+
+```inkling
+schema SimAction
+    Int32 {0, 1} shade
+end
+```
+
+The `SimAction` schema defines the `action` dictionary passed as a parameter to the `advance` method of the Python simulator. For example: `shade` == night, off, day.
+
+```inkling
+schema SimConfig
+    Int32{-1} unused
+end
+```
+
+The `SimConfig` schema in this case is not used (but is still required to be defined in Inkling) but it would define the dictionary passed as a parameter to the `set_properties` method of the Python simulator.
+
+###### Concept
+
+```inkling
+concept my_concept is classifier
+   predicts (SimAction)
+   follows input(SimState)
+   feeds output
+end
+```
+
+This concept is named `my_concept` which predicts a `SimAction` given a `SimState`.
+
+###### Simulator
+
+```inkling
+simulator energyplus_simulator(SimConfig)
+    action (SimAction)
+    state (SimState)
+end
+```
+
+This simulator is the training source for teaching `my_concept`. The Python simulator identifies itself as 'energyplus_simulator' when it connects with the AI Engine. The following statements bind the above schemas to this simulator. To define the training relationship between the simulator and the concept we must begin by defining the simulator. `energyplus_simulator` expects an action defined in the `SimAction` schema as input and replies with a state defined in the `SimState` schema as output.
+
+###### Curriculum
+
+```inkling
+curriculum my_curriculum
+    train my_concept
+    with simulator energyplus_simulator
+    objective reward_function
+        lesson my_first_lesson
+            configure
+                constrain unused with Int32{-1}
+            until
+                maximize reward_function
+end
+```
+
+The cirriculum `my_curriculum` trains `my_concept` using `energyplus_simulator`. The BRAIN that runs this Inkling code will try to maximize the value returned from `reward_function` until you stop training. `reward_function` is a method in the Python simulator.
+
+This curriculum contains one lesson, called `my_first_lesson`. It configures the simulation, by setting a number of constraints for the state of the simulator.
+
+## Simulator File
+
+```python
+# Excerpt of simulator class from the energyplus_simulator.py file
+
+class EnergyPlusSimulator(Simulator):
+    model = ePlus85Actuator()
+    server = None
+    
+    #clientState = { 'TOut': 0., 'TZone': 0., 'SolarIrradiation': 0., 'FractionShadingOn': 0. }
```

Comment:
To make this code less confusing should we remove all of the commented out code?